Sources/TuistCacheTesting/Cache/Mocks/MockCacheStorage.swift

```diff
@@ -5,15 +5,19 @@ import TuistCache
 import TuistCore
 
 public final class MockCacheStorage: CacheStoring {
-    var existsStub: ((String) -> Bool)?
+    var existsStub: ((String) throws -> Bool)?
 
     public init() {}
 
     public func exists(hash: String) -> Single<Bool> {
```

Comment:
Uff, I'm really looking forward to the introduction of `async/await`. I always found the Combine syntax very unreadable.